crates/cli/src/utils/cmd.rs

```diff
@@ -447,14 +449,21 @@ pub async fn print_traces(
     result: &mut TraceResult,
     decoder: &CallTraceDecoder,
     verbose: bool,
+    json: bool,
+    state_changes: bool,
 ) -> Result<()> {
     let traces = result.traces.as_mut().expect("No traces found");
 
     println!("Traces:");
     for (_, arena) in traces {
         decode_trace_arena(arena, decoder).await?;
-        println!("{}", render_trace_arena_with_bytecodes(arena, verbose));
+        println!("{}", render_trace_arena_inner(arena, verbose, json, state_changes));
```

Comment:
we migrated to using `sh_println!` macro for consistency but probably out of scope for this PR, @zerosnacks wdyt?

Reply:
Yeah out of scope, will do a final cleanup once we enforce `sh_println!` macro usage w/ Clippy